### DISCUSSION TITLE
Cannot all SSH key of the trial version.

### DISCUSSION THREAD

**Discussion by umresh** [WeblateOrg/weblate#6777] [Q&A]
Hi,
   I took ssh key from project -> Access control -> API access, when I try to add it to bitbucket it throws an error saying Key is already in use. I'm trying to add hosted weblate to prod project and need your help. I cannot use http authentication for push from weblate.

Thanks

**Comment by nijel** [comment]
Please add `weblate` user to your Bitbucket project, see https://docs.weblate.org/en/latest/vcs.html#accessing-repositories-from-hosted-weblate.
